site/en/guide/checkpoint.ipynb

```diff
@@ -421,7 +421,7 @@
         "\n",
         "![Visualization of the dependency graph for the example training loop](https://tensorflow.org/images/guide/whole_checkpoint.svg)\n",
         "\n",
-        "With the optimizer in red, regular variables in blue, and optimizer slot variables in orange. The other nodes, for example representing the `tf.train.Checkpoint`, are black.\n",
+        "The optimizer is in red, regular variables — in blue, and optimizer slot variables — in orange. The other nodes, for example representing the `tf.train.Checkpoint`, are black.\n",
```

Comment:
Thanks @anirudh161. I like how you rephrased it: `Create a gradient tape with persistent=True`. I kept the "If you need to do this, do that" structure, since it's an option and I wanted to have the user involved with "you". Hopefully, this is alright:

```diff
By default, the resources held by a `GradientTape` are 
released as soon as the `GradientTape.gradient` method is called.
+ To compute multiple gradients over the same computation, 
+ create a gradient tape with `persistent=True`. 
...[explanation]...
For example:
```

cc @lamberta 

This will be reflected in PR https://github.com/tensorflow/docs/pull/1769 (the autodiff guide)

Reply:
Gotcha. I think the "final" version I had looked like this (without the em dashes):

```
The optimizer is in red, regular variables are in blue, and the optimizer slot variables are in orange.
```

Reply:
Reflected in https://github.com/tensorflow/docs/pull/1770 Thank you!